src/Sylius/Component/Core/Provider/ProductVariantsPricesProvider.php

```diff
@@ -23,8 +23,10 @@
 
 final class ProductVariantsPricesProvider implements ProductVariantsPricesProviderInterface
 {
-    public function __construct(private ProductVariantPriceCalculatorInterface $productVariantPriceCalculator)
-    {
+    public function __construct(
+        private ProductVariantPriceCalculatorInterface $productVariantPriceCalculator,
+        private LowestPriceMessageCreatorInterface $lowestPriceMessageCreator,
```

Comment:
Similar as above, add this argument with the null value as default and add a deprecation

Reply:
No longer valid